src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,11 @@
 package mate.academy.lib;
 
-public @interface Inject {
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
```

Comment:
The `@Target` annotation should specify `ElementType.FIELD` or `ElementType.METHOD` instead of `ElementType.TYPE`. This is because `@Inject` is typically used to annotate fields or methods where dependencies should be injected.